frontend/vite.config.ts

```diff
@@ -24,6 +25,12 @@ const config = ({ mode }: ConfigEnv): ReturnType<typeof defineConfig> => {
         },
       },
     },
+    resolve: {
+      alias: {
+        '@img': path.resolve(__dirname, './src/assets/img'),
+        '@assets': path.resolve(__dirname, './src/assets'),
```

Comment:
I mean use it with `@assets` instead of `@img`

Reply:
Oh, sorry, got it